README.md

```diff
@@ -291,6 +291,53 @@ spark-shell --jars /path/to/spark-exasol-connector-assembly-*.jar
+|---------------------------------------------|-----------------------------------------------------------------|----------------------|
+| [Scalatest][scalatest-link]                 | A testing tool for Scala and Java developers                    | Apache License 2.0   |
+| [Scalatest Plus][scalatestplus-link]        | An integration support between Scalatest and Mockito            | Apache License 2.0   |
+| [Mockito Core][mockitocore-link]            | A mocking framework for unit tests                              | MIT License          |
+| [Testcontainers JDBC][tcont-jdbc-link]      | Testcontainers JDBC to help create JDBC based Docker containers | MIT License          |
+| [Testcontainers Scala][tcont-scala-link]    | A Scala wrapper for testcontainers-java                         | MIT License          |
+| [Spark Testing Base][spark-testing-base]    | A library that helps to create tests for Spark applications     | Apache License 2.0   |
+
+### Compiler Plugin Dependencies
+
+These plugins help with project development.
+
+| Plugin Name                                 | Purpose                                                         | License              |
+|---------------------------------------------|-----------------------------------------------------------------|----------------------|
+| [SBT Coursier][sbt-coursier-link]           | Pure Scala artifact fetching                                    | Apache License 2.0   |
+| [SBT Wartremover][sbt-wartremover-link]     | Flexible Scala code linting tool                                | Apache License 2.0   |
+| [SBT Wartremover Contrib][sbt-wcontrib-link]| Community managed additional warts for wartremover              | Apache License 2.0   |
+| [SBT Assembly][sbt-assembly-link]           | Create fat jars with all project dependencies                   | MIT License          |
+| [SBT API Mappings][sbt-apimapping-link]     | A plugin that fetches API mappings for common Scala libraries   | Apache License 2.0   |
+| [SBT Scoverage][sbt-scoverage-link]         | Integrates the scoverage code coverage library                  | Apache License 2.0   |
+| [SBT Updates][sbt-updates-link]             | Checks Maven and Ivy repositories for dependency updates        | BSD 3-Clause License |
+| [SBT Scalafmt][sbt-scalafmt-link]           | A plugin for https://scalameta.org/scalafmt/ formatting         | Apache License 2.0   |
+| [SBT Scalastyle][sbt-style-link]            | A plugin for http://www.scalastyle.org/ Scala style checker     | Apache License 2.0   |
+| [SBT Dependency Graph][sbt-depgraph-link]   | A plugin for visualizing dependency graph of your project       | Apache License 2.0   |
+| [SBT Sonatype][sbt-sonatype-link]           | A sbt plugin for publishing Scala projects to the Maven central | Apache License 2.0   |
+| [SBT PGP][sbt-pgp-link]                     | A PGP plugin for sbt                                            | Typesafe License     |
```

Comment:
```suggestion
| [SBT PGP][sbt-pgp-link]                     | A PGP plugin for `sbt`                                          | Typesafe License     |
```

Reply:
Please check that license. Checklist for that purpose should already be available.

Reply:
Yes, I checked that. It seems very strange, since they already changed the name of company to Lightbend. I am going to open an issue in the repository.

But seems very similar to MIT license.